lookbackoptions_validation

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import numpy as np
+import pandas as pd
+import pyarrow as pa
+from qablet_contracts.timetable import py_to_ts
+from qablet_contracts.timetable import TS_EVENT_SCHEMA
+from localvol import LVMCModel
```

Comment:
instead of the locally defined lvmv model, use the LVMC model that comes from the qablet package.

```
from qablet.black_scholes.mc import LVMCModel
```

in this model you will be able to specify vol simply as a number, such as 0.3
see https://qablet-academy.github.io/intro/models/mc/#fixed-vol